server/test/Main.hs

```diff
@@ -305,3 +329,60 @@ fullyAppliedScript =
     \0b22333573466e1c00800404003c0152002333500b22333573466e3c00800404003c01122\
     \010010091326353008009498cd4015d680119a802bae00112001200112001120011220021\
     \2200120014c01021820004c010544746573740001\""
+
+fixedProtocolParameters :: ProtocolParameters
+fixedProtocolParameters =
+  ProtocolParameters
+    { protocolParamProtocolVersion = (6, 0)
+    , protocolParamDecentralization = 0 / 1
+    , protocolParamExtraPraosEntropy = Nothing -- Just $ makePraosNonce "neutral"
+    , protocolParamMaxBlockHeaderSize = 1100
+    , protocolParamMaxBlockBodySize = 98304
+    , protocolParamMaxTxSize = 16384
+    , protocolParamTxFeeFixed = 155381
+    , protocolParamTxFeePerByte = 44
+    , protocolParamMinUTxOValue = Nothing
+    , protocolParamStakeAddressDeposit = 2000000
+    , protocolParamStakePoolDeposit = 500000000
+    , protocolParamMinPoolCost = 340000000
+    , protocolParamPoolRetireMaxEpoch = 18
+    , protocolParamStakePoolTargetNum = 500
+    , protocolParamPoolPledgeInfluence = 3 / 10
+    , protocolParamMonetaryExpansion = 3 / 1000
+    , protocolParamTreasuryCut = 1 / 5
+    , protocolParamUTxOCostPerWord = Just $ Lovelace 34482
+    , protocolParamCostModels = mempty
+    , protocolParamPrices =
+        Just $
+          ExecutionUnitPrices
+            { priceExecutionSteps = 721 / 10000000
+            , priceExecutionMemory = 577 / 10000
+            }
+    , protocolParamMaxTxExUnits =
+        Just $
+          ExecutionUnits
+            { executionSteps = 10000000000
+            , executionMemory = 16000000
+            }
+    , protocolParamMaxBlockExUnits =
+        Just $
+          ExecutionUnits
+            { executionSteps = 40000000000
+            , executionMemory = 80000000
+            }
+    , protocolParamMaxValueSize = Just 5000
+    , protocolParamCollateralPercent = Just 150
+    , protocolParamMaxCollateralInputs = Just 3
+    }
+
+loadFile :: IO (Either [Text.Text] ProtocolParameters)
+loadFile =
+  do
+    contents <- ByteString.readFile "test/ogmios.json"
+    return $ decodeProtocolParameters contents
+
+testParser :: Spec
+testParser =
+  it "Testing parser of ogmios parameters" $ do
+    value <- loadFile
+    shouldBe value (Right fixedProtocolParameters)
```

Comment:
The convention we've used so far in the tests is to make `shouldBe` infix

Reply:
fixed.